boards/remote/include/board.h

```diff
@@ -25,36 +24,62 @@
 
 #include "cpu.h"
 #include "periph/gpio.h"
+#include "periph/spi.h"
 
 #ifdef __cplusplus
  extern "C" {
 #endif
 
 /**
- * @brief   LED pin definitions and handlers
+ * @name LED pin definitions
  * @{
  */
-#define LED0_PIN            GPIO_PIN(3, 2)
-#define LED1_PIN            GPIO_PIN(3, 5)
-#define LED2_PIN            GPIO_PIN(2, 3)
+#define LED_RED_GPIO        GPIO_PD5
+#define LED_GREEN_GPIO      GPIO_PD4
+#define LED_BLUE_GPIO       GPIO_PD3
+/** @} */
+/**
+ * @name Macros for controlling the on-board LEDs
+ * @{
+ */
+#define LED0_ON         gpio_set(LED_RED_GPIO)
+#define LED0_OFF        gpio_clear(LED_RED_GPIO)
+#define LED0_TOGGLE     gpio_toggle(LED_RED_GPIO)
 
-#define LED0_MASK           (1 << 2)
-#define LED1_MASK           (1 << 5)
-#define LED2_MASK           (1 << 3)
+#define LED1_ON         gpio_set(LED_GREEN_GPIO)
+#define LED1_OFF        gpio_clear(LED_GREEN_GPIO)
+#define LED1_TOGGLE     gpio_toggle(LED_GREEN_GPIO)
```

Comment:
the LED macros should toggle the pins directly if possible (as the code did before), as they are also used for timing analysis and so we save the function call overhead here.